NEWS

```diff
@@ -1,3 +1,16 @@
+Flocker 0.3.3dev4 (2015-01-27)
+==============================
+
+- Applications can now be configured with a limit on the maximum size of their volume. (FLOC-729)
+- Applications can now be configured with an automatic restart policy. (FLOC-727)
+- Docker 1.2 is now required. (FLOC-721)
```

Comment:
Actually, the minimum seems to be 1.3. 
* https://github.com/ClusterHQ/flocker/blame/master/admin/packaging.py#L192

It's not as easy to see, at a glance, the package dependencies with the omnibus packaging system.
Perhaps we should try and improve that.
I think the version got incremented when we started using a newer version of the Docker API.